lib/ProductOpener/Config2_docker.pm

```diff
@@ -118,6 +119,20 @@ $events_url = $ENV{EVENTS_URL};
 $events_username = $ENV{EVENTS_USERNAME};
 $events_password = $ENV{EVENTS_PASSWORD};
 
+%oidc_options = (
```

Comment:
We have a lot of options to move to environment here :-)

Reply:
Fixed